Week1/solution1.py

```diff
@@ -0,0 +1,11 @@
+#Write a python script to enter any number and check its prime or not.
+num=int(input('Enter a number:'))
+if num > 1:
+    for i in range(2, int(num/2)+1):
+        if (num % i) == 0:
+            print(num, "is not a prime number")

```

Comment:
can you write this statement any other way.?

Reply:
yes sir, this program can be also done in the following way-
```
n=int(input('Enter number: '))
flag = False
if n <= 1:
    print(n, "is not a prime number")
elif n > 1:
    for i in range(2, n):
        if (n % i) == 0:
            flag = True
            break
    if flag:
        print(n, "is not a prime number")
    else:
        print(n, "is a prime number")

```